### PR TITLE
Backport fix for setting crowdsale end period to 30 days

### PR DESCRIPTION
**Pull request by shteryana**
Backport fix for setting crowdsale end period to 30 days

(including the unit tests)

### CODE REVIEW

File: contracts/ADXToken.sol [hunk 1]
```diff
@@ -118,7 +118,7 @@ contract ADXToken is VestedToken {
     ownerAddress = msg.sender;
     publicStartTime = _publicStartTime;
     privateStartTime = _privateStartTime;
-    publicEndTime = _publicStartTime + 4 weeks;
+    publicEndTime = _publicStartTime + STAGE_THREE_TIME_END;
```
Comment (Ivshti) [inline contracts/ADXToken.sol]: I'd rename the variable to "CROWDSALE_DURATION"

File: test/01_crowdsale.js [hunk 2]
```diff
@@ -164,14 +165,79 @@ contract('ADXToken', function(accounts) {
...
+      })
+    }))
+  })
+
+  it('Change time to 30 days after begginning of crowdsale', () => {
+    return new Promise((resolve, reject) => {
+         web3.currentProvider.sendAsync({
+          jsonrpc: "2.0",
+          method: "evm_increaseTime",
+          params: [1*24*60*60],
```
Comment (Ivshti) [inline test/01_crowdsale.js]: should we add a +30 seconds here just to ensure we're not making the transition into a state where we can't send, only by a few milliseconds?

File: test/01_crowdsale.js [hunk 3]
```diff
@@ -164,14 +165,79 @@ contract('ADXToken', function(accounts) {
...
+    return new Promise((resolve, reject) => {
+      web3.eth.sendTransaction({
+        from: currentParticipiants[0].account,
+        to: crowdsale.address,
+        value: web3.toWei(0.1, 'ether'),
+        gas: 130000
+      })
+    }).then(function() { throw new Error('Cant be here'); })
+    .catch(function(err) {
+      assert.equal(err.message, 'VM Exception while processing transaction: invalid opcode');
```
Comment (Ivshti) [inline test/01_crowdsale.js]: how does this work? there should be a `resolve` call I think

File: test/01_crowdsale.js [hunk 4]
```diff
@@ -164,14 +165,81 @@ contract('ADXToken', function(accounts) {
...
+    const currentParticipiants = participiants.slice(0, 3)
+
+    return new Promise((resolve, reject) => {
+      web3.eth.sendTransaction({
+        from: currentParticipiants[0].account,
+        to: crowdsale.address,
+        value: web3.toWei(0.1, 'ether'),
+        gas: 130000
+      },(err, result) => {
+        err? reject(err) : resolve()
```
Comment (NikolaBorislavovHristov) [inline test/01_crowdsale.js]: err? reject(err) : reject(new Error('Cant be here'))

**Pull request closed without merging**